code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -143,3 +146,117 @@
+				if("blue")
+					new /obj/item/clothing/suit/armor/knight/blue(get_turf(H))
+					new /obj/item/clothing/head/helmet/knight/blue(get_turf(H))
+				if("red")
+					new /obj/item/clothing/suit/armor/knight/red(get_turf(H))
+					new /obj/item/clothing/head/helmet/knight/red(get_turf(H))
+				if("templar")
+					new /obj/item/clothing/suit/armor/knight/templar(get_turf(H))
+					new /obj/item/clothing/head/helmet/knight/templar(get_turf(H))
+
+			switch (randomizeswords)
+				if("unlucky") //so the chance to get an unlucky item does't clutter the main pool of swords
+					var/noluck = pick(/obj/item/weapon/kitchen/utensil/knife/plastic, /obj/item/weapon/screwdriver, /obj/item/weapon/wirecutters, /obj/item/toy/foamblade, /obj/item/toy/sword)
+					new noluck(get_turf(H))
+				if("misc")
+					var/miscpick = pick(/obj/item/weapon/scythe, /obj/item/weapon/harpoon, /obj/item/weapon/sword, /obj/item/weapon/sword/executioner, /obj/item/weapon/claymore, /obj/item/weapon/sord)
+					new miscpick(get_turf(H))
+				if("glass")
+					if(prob(50))
+						new /obj/item/weapon/shard(get_turf(H))
+					else
+						new /obj/item/weapon/shard/plasma(get_turf(H))
+				if("hatchet")
+					new /obj/item/weapon/hatchet(get_turf(H))
+				if("armblade") // good luck getting it off. Maybe cut your own arm off :^)
+					new /obj/item/weapon/armblade(get_turf(H))
+				if("pickaxe")
+					var/pickedaxe = pick(/obj/item/weapon/pickaxe, /obj/item/weapon/pickaxe/silver, /obj/item/weapon/pickaxe/gold, /obj/item/weapon/pickaxe/diamond)
+					new pickedaxe(get_turf(H))
+				if("pcutter")
+					new /obj/item/weapon/pickaxe/plasmacutter(get_turf(H))
+				if("esword")
+					new /obj/item/weapon/melee/energy/sword(get_turf(H))
+					if(prob(70)) //chance for a second one to make a double esword
+						new /obj/item/weapon/melee/energy/sword(get_turf(H))
+				if("alt-esword")
+					if(prob(75))
+						new /obj/item/weapon/melee/energy/sword/pirate(get_turf(H))
+						if(prob(70))
+							new /obj/item/weapon/melee/energy/sword/pirate(get_turf(H))
+					else //hope you're the clown
+						new /obj/item/weapon/melee/energy/sword/bsword(get_turf(H))
+						if(prob(70))
+							new /obj/item/weapon/melee/energy/sword/bsword(get_turf(H))
+				if("machete")
+					new /obj/item/weapon/melee/energy/hfmachete(get_turf(H))
+					if(prob(70))
+						new /obj/item/weapon/melee/energy/hfmachete(get_turf(H))
+				if("kitchen")
+					if(prob(60))
+						if(prob(25))
+							new /obj/item/weapon/kitchen/utensil/knife/large(get_turf(H))
+						else
+							new /obj/item/weapon/kitchen/utensil/knife/large/butch(get_turf(H))
+					else
+						new /obj/item/weapon/kitchen/utensil/knife/large/butch/meatcleaver(get_turf(H))
+				if("spear")
+					if(prob(50))
+						new /obj/item/weapon/spear(get_turf(H))
+					else
+						new /obj/item/weapon/melee/lance(get_turf(H))
+				if("katana") //feeling lucky?
+					if(prob(5))
+						new /obj/item/weapon/katana/hfrequency(get_turf(H))
```

Comment:
"no"

Reply:
The key question here is "Is an incredibly broken item offset by incredible rarity?"
I'm not sure that it is, at this level of broken. The HF sword is an instakill, so while it may only show up in 1% of rounds, it is nearly guaranteed to ruin those 1% of rounds for everyone except the guy who gets it. In which case, why include it at all?
I don't think the HF sword as it exists right now should ever be wielded by anyone except at the will of an admin.
If you wish to include it in the summoned swords, I'd recommend making a weaker version, perhaps coding Zandatsu so that the perfect sharpness is only active in short bursts with long cooldown periods.

Reply:
While I agree it's op in all ways, I feel that the 0.25% chance to get it is good. As for combat use, the spell doesn't disable ranged combat choices. Tazers, lasers, and other guns exist still and as well all know it's stun and done.

Reply:
0.25% chance per player means 11.5% chance in a 50 player run.

Reply:
I agree, nuke it

Reply:
I guess it's fair enough to remove it so babies don't complain since they don't know how to pick up a taser.